src/schema/v2/artwork/index.ts

```diff
@@ -136,6 +136,8 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
       artworkLocation: {
         type: GraphQLString,
         resolve: (artwork) => artwork.artwork_location,
+        description:
+          'Represents the location of the artwork for "**My Collection**" artworks',
```

Comment:
I wonder if `**My Collection**` is necessary here or if `"My Collection"` would be easier to read 🤔

Reply:
Haha. I just wanted to give more prominence. I will change it now.